internal/app/repository/sqlite/methods.go

```diff
@@ -0,0 +1,107 @@
+package sqlite
+
+import (
+	"github.com/PhilippNikitin/go_final_project/internal/app/model"
+)
+
+// Сохранение задачи в базу данных
+func (repo *TaskRepository) Create(task model.Task) (int64, error) {
+	res, err := repo.DB.Exec(
+		`INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)`,
+		task.Date, task.Title, task.Comment, task.Repeat)
+	if err != nil {
+		return 0, err
+	}
+
+	// Получение ID созданной задачи
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
```

Comment:
Хорошо бы все ошибки оборачивать, тогда будет проще искать места возникновения
```suggestion
		return 0, fmt.Errorf("erorr while getting inserted id %w", err)
```